www/docs/client/react/useSubscription.md

```diff
@@ -55,7 +59,7 @@ type TRPCSubscriptionResult<TOutput, TError> = {
    * The current status of the subscription.
    * Will be one of: `'idle'`, `'connecting'`, `'pending'`, or `'error'`.
    *
-   * - `idle`: subscription is disabled
+   * - `idle`: subscription is disabled or ended
```

Comment:
a double meaning might be something worth splitting up, imagine someone wanting to restart the subscription with some new parameters or just restart for whatever reason. Having a distinct "complete" state might be useful

Reply:
it's completed if `enabled && status === 'idle`

```ts
function MySub() {
  const [enabled, setEnabled] = React.useState(false)
  const sub = trpc.sub.useSubscription({}, { enabled, })
  const completed = sub.status === 'idle' && !enabled
```

but yes, might be good to add.